lib/plugins/query.js

```diff
@@ -14,12 +14,16 @@ var assert = require('assert-plus');
  * @public
  * @function queryParser
  * @param    {Object}   options an options object
+ * @param    {Object}   qsOptions an options object for qs
  * @returns  {Function}
  */
-function queryParser(options) {
+function queryParser(options, qsOptions) {
 
     var opts = options || {};
+    var qsOpts = qsOptions || {};
```

Comment:
It doesn't look like `qsOpts` is used in `parseQuerystring`. Is that intentional?